src/Illuminate/Encryption/OpenSSLEncrypter.php

```diff
@@ -55,19 +50,33 @@ public static function supported($key, $cipher)
     {
         $length = mb_strlen($key, '8bit');
 
-        return ($cipher === 'AES-128-CBC' && $length === 16) ||
-               ($cipher === 'AES-256-CBC' && $length === 32);
+        return ($cipher === self::AES_128 && $length === 16) ||
```

Comment:
IMO, it would be cleaner to also store the key lengths in a constant:

```php
const KEY_LENGTHS = [
    self::AES_128 => 16,
    self::AES_256 => 32,
];
```

Reply:
That would be much clearer. I'll add that.